toolkit/scripts/check_entangled_specs.py

```diff
@@ -35,6 +35,7 @@
 version_matching_groups = [
     frozenset([
         "SPECS/hyperv-daemons/hyperv-daemons.spec",
+        "SPECS/kernel-hci/kernel-hci.spec",
```

Comment:
Just to double-check expectations - By adding this line, the PR gate is updated to flag that `kernel-hci` must be automatically updated alongside our normal `kernel` updates. Meaning if `kernel-hci` doesn't build (i.e., patch not applying cleanly), we will have to hold the normal `kernel` update until `kernel-hci` is building/working successfully.

Is that the intention? Have we worked out what will happen if the mlx5 patches do not apply cleanly on a kernel version update to get it resolved in an expedient timeframe?

Reply:
No, the intention is not to block the kernel package especially to address critical CVEs. Perhaps this check can be ignored in the unlikely event kernel-hci patch application fails.

Reply:
is it guaranteed that kernel-hci will/should always be at same version number than kernel ? Should we linked the 2 ?

Reply:
I've removed the entanglement.